src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -328,4 +333,49 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     PolicyResponse response = new PolicyResponse().policies(Collections.singletonList(policy));
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
+
+  private void validateIngestParams(IngestRequestModel ingestRequestModel, UUID datasetId) {
+    Dataset dataset = datasetService.retrieve(datasetId);
+    CloudPlatformWrapper platform =
+        CloudPlatformWrapper.of(dataset.getDatasetSummary().getStorageCloudPlatform());
+
+    if (platform.isAzure()) {
+      validateAzureIngestParams(ingestRequestModel);
+    }
+  }
+
+  private void validateAzureIngestParams(IngestRequestModel ingestRequest) {
+    List<String> errors = new ArrayList<>();
```

Comment:
I vaguely remember discussing throwing an error if the user tries to set the "csv_allow_quoted_newlines"  or "csv_null_marker" for Azure ingest since we don't support them for Azure. Could we include this here?

Reply:
Oh I talked about this with Muscles. We provide defaults for those two fields, so it would be impossible to tell if a user defined a value that is the same as the default, or if it was just the default that we provide. Since we can't determine whether its a user action or not, we decided that we should just document that the fields will be ignored for Azure.